### ISSUE TITLE
Consider renaming this tool

### ISSUE THREAD

**Issue by Ammarpad** [wikimedia/WikiContrib#105]
Hello everyone,

For sometime now, I've been wondering why this tool was named the way it's. I believe this tool is clearly misnamed. I am not sure whether there was any conscious decision/deliberation in choosing the name or if it just happened to be chosen by the primary developer in the initial phase of the code.

If there was any discussion endorsing the suitability of this name then you can just close this issue, otherwise I suggest to rename it. 

The tool does not track any "wiki" activity nor wiki "contrib[ution]" and so its name should not inaccurately suggest that it do.

The tool express purpose is to track and showcase "developer activity/contrib," and so the name should reflect that. I don't have great suggestions though, but I believe something like "DeveloperStats" would be better and more accurate than what we have now.  

"DeveloperContrib" and the like are also equally viable alternatives as well as other suggestions people might have.

**Comment by srish** [comment]
@Ammarpad Thanks for sharing your concern and sorry for the late response! I would like to share that WikiContrib is the second time we changed the name of the project. After we announced it on Wikitech, there were some concerns shared on the first name, and we decided to change it to WikiContrib: https://lists.wikimedia.org/pipermail/wikitech-l/2019-June/092169.html

Also, this tool is going to be not just about tracking Gerrit and Phabricator but also other technical contributions some of them are listed here: 
https://github.com/wikimedia/WikiContrib/blob/master/README.md#planned-features. So, soon it will track broader contributions in the Wikimedia world :)
